### ISSUE TITLE
evaluate accessibility of buttons with white text

### ISSUE THREAD

**Issue by evanwill** [uidaholib/main-web-redesign_draft#372]
i think white text on pride gold and clearwater is not enough contrast

**Comment by evanwill** [comment]
@dcnb @owikle 

These buttons fail accessibility audits due to low color contrast:
 
- .btn-pride-gold (like the "go" search button on index) 
- .btn-clearwater (like "view VIVO Profile" on featured researchers on index)
- #bigMenu (the big MENU dropdown button) 

I think we should make #bigMenu color black. 
I am not sure what to do with the buttons. 
I think .btn-pride-gold text is pretty hard to read when the button is on white background, less so with a black background as the "Go" button.
